qulice-checkstyle/src/main/java/com/qulice/checkstyle/MethodsOrderCheck.java

```diff
@@ -82,7 +82,7 @@ private void checkClass(final DetailAST node) {
     /**
      * Checks order of methods.
      *
-     * @param methods Iterable<DetailAST> methods
+     * @param methods Nodes representing class methods
```

Comment:
@vkuchyn it's OK, but I see that you didn't know how to make proper `@code` tag. It should be just `{@code Iterable<DetailAST>}`